extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -210,14 +210,52 @@ function tests(name) {
           resizable: '',
         }, 'https://schema.org').then(element => {
           const impl = element.implementation_;
-          const attemptChangeSizeSpy = sandbox.spy(impl, 'attemptChangeSize');
-          impl.apiHandler_.updateSize_(217, 114);
-          expect(attemptChangeSizeSpy.callCount).to.equal(1);
-          expect(attemptChangeSizeSpy.firstCall.args[0]).to.equal(217);
-          expect(attemptChangeSizeSpy.firstCall.args[1]).to.equal(114);
+          const attemptChangeSize = sandbox.stub(impl, 'attemptChangeSize',
```

Comment:
Don't quite understand what this test is trying to do here.
The name suggests it's to test a fallback is called when resize has overflow. But it's unclear what is the fallback.

Also, I saw the most logic is around the internal member field: `apiHandler_`. If the logic we're testing is inside `apiHandler_`, should we just move the test to `apiHandler_`'s unit test?